### PR TITLE
Create BLorient9798 (Sawāsəw)

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
Some questions concerning  probable new IDs for ms_i2, ms_i3 and ms_i4 are still under discussion in https://github.com/BetaMasaheft/Documentation/issues/2580 and in https://github.com/BetaMasaheft/Documentation/issues/2582. 
For ms_i1.2 it was decided not to create an individual ID and to nest it to the Sawāsəw. Cf. https://github.com/BetaMasaheft/Documentation/issues/2578.

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+                                the Emperor <persName ref="PRS9430Tewodros">Theodore (Tewodros) (1860 EC, i.e. 1868 AD)</persName>.
+                                That event is mentioned under the year '523' based on the Era of the Martyrs, which should however be corrected to
+                                <date when-custom="521" calendar="grace" when="1868">521</date> to correspond to the year of Tewodros' death.
+                            </note>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="57v" to="58r"/>
+                            <title ref="LIT0000000000000000000">Qǝne poems attributed to Gabra Masiḥ</title>
+                            <note>Nine <ref type="authFile" corresp="#Qene">qǝne songs</ref> attributed to person with the name
```
Comment (karljonaskarlsson) [inline LondonBritishLibrary/orient/BLorient9798.xml:132]: Here as well

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+                        </msItem>
+                        <msItem xml:id="ms_i4">
+                            <locus from="57v" to="58r"/>
+                            <title ref="LIT0000000000000000000">Qǝne poems attributed to Gabra Masiḥ</title>
+                            <note>Nine <ref type="authFile" corresp="#Qene">qǝne songs</ref> attributed to person with the name 
+                                <persName ref="PRS14499GabraMasih" role="author">Gabra Masiḥ</persName>: 
+                                5 <ref type="authFile" corresp="#EtanaMogar"><foreign xml:lang="gez-tr">ʿǝṭāna mogar</foreign></ref>, 
+                                1 <ref type="authFile" corresp="#Wazema"><foreign xml:lang="gez-tr">wāzemā</foreign></ref>, 
+                                1 <ref type="authFile" corresp="#Sellase"><foreign xml:lang="gez-tr">śǝllāse</foreign></ref>, 
+                                and 1 in the <foreign xml:lang="gez">ጉባዔ፡ ቃና፡</foreign>-type,
```
Comment (karljonaskarlsson) [inline LondonBritishLibrary/orient/BLorient9798.xml:137]: We should probably create an authority file for the qǝne type Gubāʾe Qānā.

It's one of the types mentioned in lists, for example in the EAe (s.v. qǝne).

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+                                    <date when-custom="521" calendar="grace" when="1868">521</date> or shortly after.</desc>
+                                <date when="1868">1868</date>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#49rb"/>
+                                    <desc type="GuestText">Calculation of the auspicious and ill-omened days 
+                                        (<ref type="work" corresp="#LIT4145Onausp">Ḥasāba ṣǝnʿǝt wa-ṣǝnṣǝnǝt</ref>).</desc>
```
Comment (karljonaskarlsson) [inline LondonBritishLibrary/orient/BLorient9798.xml:188]: I would suggest ḥassāba ṣǝnǝʿt wa-ṣǝnṣǝnt

**Review by karljonaskarlsson — CHANGES_REQUESTED** [review]

**Review by karljonaskarlsson — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+                    </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="39vb"/>
+                            <title ref="LIT2293Sawase">Sawāsǝw</title>
+                        <msItem xml:id="ms_i1.1">
+                            <locus from="1r" to="36vb"/><locus from="39ra" to="39va"/>
+                            <title>Sawāsǝw (wordlist)</title>
+                            <note>List of Gǝʿǝz-words and their Amharic translations similar to the one in manuscript BnF d'Abbadie 223.</note>
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient9798.xml:39]: Yes. For this reason, it is useful to create a stub with the ID BNFabb223. I think, Denis is working on the Abbadie-collection. @DenisNosnitsin1970 : Can I create a stub for BNFabb223 in a new pull request or is Abbadie 223 in your processing at the moment?

File: LondonBritishLibrary/orient/BLorient9798.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient9798.xml]: It is not listed among the seven modes that are listed as "Qene types" in the Qene-related files:
https://betamasaheft.eu/authority-files/list?keyword=Qene
As well, I did not find it, when looking up ጉባዔ፡ or ቃና. So, I think, it does not exist, yet.

File: LondonBritishLibrary/orient/BLorient9798.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient9798.xml:188]: OK.

File: LondonBritishLibrary/orient/BLorient9798.xml (hunk 2 above)
Comment (karljonaskarlsson) [reply LondonBritishLibrary/orient/BLorient9798.xml]: Yes, then creating it would probably be a good idea. Or what do you say, @DenisNosnitsin1970 ?

**Review by karljonaskarlsson — APPROVED** [review]

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 5]
```diff
@@ -0,0 +1,281 @@
...
+                                    በፍጻሜ፡ ግፍዕከሰ፡ ከመ፡ ቃለ፡ ሙሴ፡ ጸሐፈ።
+                                    ዘመን፡ ለከ፡ እመ፡ ኃለፈ።
+                                    አሐዱ፡ ወልድከ፡ ያስተጋብእ፡ ጽንፈ።
+                                    ወክልኤቱ፡ ይሰድዱ፡ እልፈ።</incipit>
+                            </msItem>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <locus from="50r" to="56v"/>
+                            <title ref="LIT0000000000000000000">Calendar of the movable feasts (Ḥasaba ḥǝgg)</title>
+                            <note>A kind of perpetual calendar based on the great lunar cycle of 532 years. The columns contain:
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient9798.xml:116]: Please create first the all necessary LIT records.

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 6]
```diff
@@ -0,0 +1,281 @@
...
+                                8 <ref type="authFile" corresp="#Wazema"><foreign xml:lang="gez-tr">wāzemā</foreign></ref>,
+                                4 <ref type="authFile" corresp="#EtanaMogar"><foreign xml:lang="gez-tr">ʿǝṭāna mogar</foreign></ref> 
+                                and 1 <ref type="authFile" corresp="#Zayeeze"><foreign xml:lang="gez-tr">zayǝʾǝzeni</foreign></ref>.</note>
+                            <incipit xml:lang="gez">ናሁ፡ እጽሕፍ፡ መዝገበ፡ ድርሳን፡ ዘይረስዎ፡ አዕሩግ፡ የዋሃን፡ ወወራዙት፡ ማእምራን፡ ውስተ፡ ቅድስት፡ ቤተ፡ ክርስቲያን፡
+                                ባሕቱ፡ ይክሥ<pb n="40ra"/>ት፡ ሊተ፡ እግዚአብሔር፡ ጥበበ፡ ወኪነ፡ አሜን።</incipit>
+                            <textLang mainLang="gez"/>
+                            <msItem xml:id="ms_i2.1">
+                                <title ref="LIT7038Qene">Qǝne poem of the <foreign xml:lang="gez">śǝllāse</foreign>-type on ʾEwosṭātewos</title>
+                                <note>Probably on <persName ref="PRS3934Ewostate"/>.</note>
+                                <incipit xml:lang="gez"><persName ref="PRS3934Ewostate">ኤዎስጣቴዎስ፡</persName> እንዘ፡ እልፍ፡ ውሉድ፡ ውስተ፡ ሕፅንከ፡
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient9798.xml:104]: please include cert="...". Actually, we cannot be sure that this is St Ewostatewos from norther Ethiopia.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 7]
```diff
@@ -0,0 +1,281 @@
...
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Neat handwriting, possibly several hands.</desc>
+                                <date notBefore="1700" notAfter="1868" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                            <handNote xml:id="h2" script="Ethiopic">
+                                <locus target="#56v"/>
+                                <desc>Notes in a Ḥasaba ḥǝgg, which notes the death of Emperor Tewodros and must be therefore in the year 
+                                    <date when-custom="521" calendar="grace" when="1868">521</date> or shortly after.</desc>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient9798.xml:179]: The verb "record" would work out better, and: "notes record" or "Hassaba hegg" records? I would write always Tewodros II, and I personally use always designation "king".

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 8]
```diff
@@ -0,0 +1,281 @@
...
+                                    <locus target="#1r"/>
+                                    <desc type="OwnershipNote">In the blank space between the columns a note partly illegible with the name of an 
+                                        British army officer as owner of the manuscript.</desc>
+                                    <q xml:lang="en"><persName role="owner" ref="PRS14353Edwards">William Wilson Edwards</persName>, 
+                                        Capt. of Madras Light Cavalry and Abyssinian Transport Train 
+                                        <date notBefore="1867" notAfter="1868">1867-1868</date></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#57r"/>
+                                    <desc type="OwnershipNote">Note indicating, that this book belonged to Emperor
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient9798.xml:205]: One , is not needed here, please use the numbers with the name of the king, for the sake of conviniency.

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 9]
```diff
@@ -0,0 +1,281 @@
...
+                                        Capt. of Madras Light Cavalry and Abyssinian Transport Train 
+                                        <date notBefore="1867" notAfter="1868">1867-1868</date></q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#57r"/>
+                                    <desc type="OwnershipNote">Note indicating, that this book belonged to Emperor 
+                                        <persName role="owner" ref="PRS9142TaklaGi">Takla Giyorgis</persName>, 
+                                        who gave it to his sons.</desc>
+                                    <q xml:lang="gez">ዝሰዋስው፡ ዘ<persName role="owner" ref="PRS9142TaklaGi">ተክለ፡ ጊዮርጊስ፡ 
+                                        <roleName type="function">ንጉሥ፡</roleName></persName>  ዘወሀቦሙ፡ ለውሉዱ፡</q>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient9798.xml:209]: Space before ዘወሀቦሙ፡.

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 10]
```diff
@@ -0,0 +1,281 @@
...
+                                </item>
+                                <item xml:id="e3">
+                                    <locus target="#57r"/>
+                                    <desc type="OwnershipNote">Note indicating, that this book belonged to a person with the name 
+                                        <persName role="owner" ref="PRS14501Yohannes">Yoḥannǝs</persName></desc>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient9798.xml:214]: One , too much.

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 11]
```diff
@@ -0,0 +1,281 @@
...
+                                <item xml:id="e4">
+                                    <locus target="#29ra #57r"/>
+                                    <desc type="Unclear">Scribbles and pen trials in red ink</desc>
+                                </item>
+                            </list>
+                        </additions>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1700" notAfter="1868">18th/19th century</origDate>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient9798.xml:226]: Do we accept the information that this book is of Takla Giyorgis and was produced not later than his reign or before that? If so, the dating should be modified.

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 12]
```diff
@@ -0,0 +1,281 @@
...
+                                <desc>Notes in a Ḥasaba ḥǝgg, which notes the death of Emperor Tewodros and must be therefore in the year 
+                                    <date when-custom="521" calendar="grace" when="1868">521</date> or shortly after.</desc>
+                                <date when="1868">1868</date>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#49rb"/>
+                                    <desc type="GuestText">Calculation of the auspicious and ill-omened days
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient9798.xml:187]: "GuestText" in additional notes is encoded in a slightly different way. I think you have to structure it like, for instance: 
`<desc type="GuestText"><title type="incomplete" ref="LIT1104Anapho"/>An excerpt from the Anaphora of the Apostles</desc>` (here and below)

File: LondonBritishLibrary/orient/BLorient9798.xml [hunk 13]
```diff
@@ -0,0 +1,281 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="BLorient9798" xml:lang="en" type="mss">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Sawāsǝw, calendar for the movable feasts, Amharic explanation of the creed</title>
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient9798.xml:8]: Calendar

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: LondonBritishLibrary/orient/BLorient9798.xml (hunk 11 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient9798.xml:226]: Yes, I would accept and modified the information.

File: LondonBritishLibrary/orient/BLorient9798.xml (hunk 12 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient9798.xml:187]: I changed it for a1, but for a2, I have no LIT and no information to identify the work.
```

<item xml:id="a1">
                                    <locus target="#49rb"/>
                                    <desc type="GuestText"><title type="incomplete" ref="LIT4145Onausp"/>Calculation of the auspicious and 
                                        ill-omened days</desc>
                                    <q xml:lang="gez"><title>ሐሳበ፡ ተአምር፡ ጽንዕት፡ ወጽንጽንት፡</title> በመስከረም፡ ፲ወ፪ጽንዕት። ወላዕሉ፡ ጽንፅንት። በጥቅምት፡ ፳፫ጽንፅንት፡ 
                                        ወላዕሉ፡ ጽንዕት፡</q>
                                </item>
                                <item xml:id="a2">
                                    <locus target="#49v"/>
                                    <desc type="GuestText">A fragment of an Amharic work concerning the Trinity.</desc>
                                </item>
```

File: LondonBritishLibrary/orient/BLorient9798.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/orient/BLorient9798.xml:39]: I am creating the record, but it is probably Abb. 146 (223 is the number in the catalogue).

[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient9798.xml]: I added the lD to BNFabb146.

File: LondonBritishLibrary/orient/BLorient9798.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply LondonBritishLibrary/orient/BLorient9798.xml]: I found [Gubāʾe qānā](https://betamasaheft.eu/authority-files/Gubaeqana/main?hi=Gubae) as an authority file.

**Comment by CarstenHoffmannMarburg** [comment]
@DenisNosnitsin1970: I think all requests have been solved. Correct?

**Comment by CarstenHoffmannMarburg** [comment]
> @DenisNosnitsin1970: I think all requests have been solved. Correct?

@eu-genia @thea-m: Can you merge this branch? I think, I can not overrun the request by Denis and merge it by myself.

**Pull request merged**